settings/installer/packstack/network/variant/_neutron_gre.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: gre
+
+config:
+    CONFIG_NEUTRON_OVS_TENANT_NETWORK_TYPE: "{{ !lookup installer.network.variant }}"
```

Comment:
where the 'installer.network.variant' will come from? can't find it in case of using neutron network